src/js_crypto/bigint256.ts

```diff
@@ -0,0 +1,85 @@
+/* global joo_global_object, caml_js_to_bool, caml_jsstring_of_string, caml_string_of_jsstring,
```

Comment:
the files in `js_crypto` are just dumped here to modify in the next PR -- please ignore